File: CS-Examples/09_Charts/SetChartBackgroundColor.cs
using System;
using System.Windows.Forms;
using Spire.Xls;

namespace SetChartBackgroundColor
{

	public partial class Form1 : Form
	{
        public Form1()
        {
            InitializeComponent();
        }

		private void btnRun_Click(object sender, EventArgs e)
		{
            //Create a workbook
			Workbook workbook = new Workbook();

            //Load the document from disk
            workbook.LoadFromFile(@"..\..\..\..\..\..\Data\ChartSample1.xlsx");

            //Get the first worksheet from workbook and then get the first chart from the worksheet
            Worksheet ws = workbook.Worksheets[0];
            Chart chart = ws.Charts[0];

            //Set background color
            chart.ChartArea.ForeGroundColor = System.Drawing.Color.LightYellow;

            //Save the document
            string output = "SetChartBackgroundColor.xlsx";
			workbook.SaveToFile(output, ExcelVersion.Version2013);

            //Launch the file
			ExcelDocViewer(output);
		}
        private void ExcelDocViewer(string fileName)
        {
            try
            {
                System.Diagnostics.Process.Start(fileName);
            }
            catch { }
        }

        private void btnClose_Click(object sender, EventArgs e)
        {
            Close();
        }

	}
}

File: CS-Examples/09_Charts/SetChartBackgroundColor.vb
Imports Spire.Xls

Namespace SetChartBackgroundColor

	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub btnRun_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnRun.Click
            ' Create a new instance of Workbook
            Dim workbook As New Workbook()

            ' Load the Excel file from the specified path
            workbook.LoadFromFile("..\..\..\..\..\..\Data\ChartSample1.xlsx")

            ' Get the first worksheet in the workbook
            Dim ws As Worksheet = workbook.Worksheets(0)

            ' Get the first chart in the worksheet
            Dim chart As Chart = ws.Charts(0)

            ' Set the foreground color of the chart area to LightYellow
            chart.ChartArea.ForeGroundColor = Color.LightYellow

            ' Save the modified workbook to a new file
            Dim output As String = "SetChartBackgroundColor.xlsx"
            workbook.SaveToFile(output, ExcelVersion.Version2013)
            ' Release the resources used by the workbook
            workbook.Dispose()
            'Launch the file
            ExcelDocViewer(output)
		End Sub
		Private Sub ExcelDocViewer(ByVal fileName As String)
			Try
				Process.Start(fileName)
			Catch
			End Try
		End Sub

		Private Sub btnClose_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnClose.Click
			Close()
		End Sub

	End Class
End Namespace
